src/homepageExperience/components/steps/python/ExecuteQuerySql.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+import {event} from 'src/cloud/utils/reporting'
+
+const logCopyCodeSnippet = () => {
+  event('firstMile.pythonWizard.executeQuery.code.copied')
+}
+
+type OwnProps = {
+  bucket: string
+}
+
+export const ExecuteQuerySql = (props: OwnProps) => {
+  const {bucket} = props
+
```

Comment:
This is okay, but I might suggest moving the SQL snippets and queries into a separate file so that they're not mixed in with JSX.

Reply:
This was in keeping with the existing pattern.